src/androidTest/java/com/mixpanel/android/mpmetrics/MixpanelNotificationBuilderTest.java

```diff
@@ -85,6 +99,14 @@ public void testNoTitle() {
         verify(builderSpy).setContentTitle(DEFAULT_TITLE);
     }
 
+    public void testSubTitle() {
```

Comment:
We already have a test for this below called `testSubText`

Reply:
Deleted